src/theory/bv/bitblast/simple_bitblaster.h

```diff
@@ -53,6 +53,8 @@ class BBSimple : public TBitblaster<Node>
   /** Create 'bits' for variable 'var'. */
   void makeVariable(TNode var, Bits& bits) override;
 
+  /** Add d_variables to termSet. */
```

Comment:
Can you give justification for why this is necessary here? I.e. that BV_EAGER_ATOM has terms that may appear beneath ITE.